apps/api/src/app/workflows/usecases/delete-notification-template/delete-notification-template.usecase.ts

```diff
@@ -94,6 +96,19 @@ export class DeleteNotificationTemplate {
           changeId: parentChangeId,
         })
       );
+
+      this.analyticsService.track(`Removed Notification Template`, command.userId, {
+        _organization: command.organizationId,
+        _environment: command.environmentId,
+        _templateId: command.templateId,
+        data: {
+          createdAt: item.createdAt,
+          lastUpdatedAt: item.updatedAt,
+          deletedAt: new Date(),
```

Comment:
No need to pass this, as the time is already created by segment and downstream providers